packages/slice-machine/components/UpdateVersionModal/index.tsx

```diff
@@ -31,7 +31,7 @@ const UpdateVersionModal: React.FC = () => {
     useSliceMachineActions();
 
   React.useEffect(() => {
-    if (updateVersionInfo.updateAvailable) openUpdateVersionModal();
+    if (updateVersionInfo.updateAvailable && !isOpen) openUpdateVersionModal();
```

Comment:
I will investigate 🧐

Reply:
I took a look to the Github history and there is no back-and-forth with this isOpen status. I think we you are confusing with the fact that the updateVersionModal popped up too many times. This is not related to that, it's just to avoid emitting actions when the modal is already open :)

Reply:
k